### DISCUSSION TITLE
Failed devtools::check()

### DISCUSSION THREAD

**Discussion by Cateline** [JRaviLab/MolEvolvR#104] [Q&A]
Hi team, @jananiravi @the-mayer @falquaddoomi @epbrenner 
I'm having issues running the devtools check for my code, which I believe is subsequently causing the pkgdown fail in my pull request. This is the error message I am getting. I'd really appreciate your help in resolving this issue
![image](https://github.com/user-attachments/assets/1559dacf-34f4-4f17-bdc6-ef54b8baa7b1)

**Comment by jananiravi** [comment]
moved to #105
